ChaosMod/Effects/db/Peds/PedsEveryoneWepController.cpp

```diff
@@ -127,4 +127,23 @@ static RegisterEffect registerEffect7(EFFECT_EVERYONE_BATTLEAXE, OnStartBattleAx
 		.Id = "peds_battleaxe",
 		.EEffectGroupType = EEffectGroupType::Weapons
 	}
+);
+
+static void OnStartFlareGun()
+{
+	static const Hash flareGunHash = GET_HASH_KEY("WEAPON_FLAREGUN");
+
+	for (Ped ped : GetAllPeds())
+	{
+		GIVE_WEAPON_TO_PED(ped, flareGunHash, 9999, true, true);
+		SET_CURRENT_PED_WEAPON(ped, flareGunHash, true);
```

Comment:
the second "true" is "equip now", so the second native should not be necessary

Reply:
I just copied the code from all the other give everyone effects. But I can change it, when I get back to my pc.